iree/tools/BUILD

```diff
@@ -238,7 +238,6 @@ cc_library(
         "@llvm-project//mlir:Translation",
         "@org_tensorflow//tensorflow/compiler/mlir/xla:xla_dialect_registration",
     ] + IREE_COMPILER_DIALECT_MODULES + IREE_COMPILER_TARGET_BACKENDS,
-    alwayslink = 1,
```

Comment:
I think removing alwayslink here isn't right. iree_translate_main has the main function so it needs to always be linked in. This doesn't have to do with dialect registration

Reply:
Okay, I'll drop the two commits that touch the iree/tools configs.